### ISSUE TITLE
pombe annotation to delete ARUK-UCL from PMID:28242692

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1631]
This paper has been fully curated by  the author and myself.
https://curation.pombase.org/pombe/curs/83bb6f203d9fe65a/ro
and fully approved by the author.

vps4
http://www.uniprot.org/uniprot/Q09803
Please delete all of the annotations  from  ARUK-UCL
They are very indirect phenotypes.

vps4 has perforated nuclear envelope but it isn't involved in "the regulation of nuclear permeability", and it isn't involved in "spindle pole body separation"- spindle pole body separation is only  affected in the mutant because the nuclear enveloper is buggered and  the  SPB needs to be embedded in the nuclear envelope. 

In a normal cell vps4 does not  participate in the process of spindle pole body separation.

Similarly lem2
http://www.uniprot.org/uniprot/Q10109

Please delete 
negative regulation of cell growth (phenotype)
spindle pole body separation (this protein is upstream of spindle pole body separation)
   
Thanks!

  @RLovering 

val

**Comment by ValWood** [comment]
All the ER organization stuff needs deleting too.
The ER related stuff is due to "karmella formation"
https://github.com/pombase/fypo/issues/3013
This is not a normal process , it is pathogenic in Fungi.


I'm checking my archive and I now see that after I finished  I sent this paper  to you to curate the human part.....

This is  really good example of where  phenotypes  do not == GO processes.....

**Comment by Antonialock** [comment]
@RLovering 
offtopic but I also spotted that the parkinsons project has annotated mou
I spotted that the parkinsons project annotated mouse Parkn to:

negative regulation of endoplasmic reticulum stress-induced neuron intrinsic apoptotic signaling pathway
and 
positive regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway

The abstract for the +ve reg paper begins:
"Parkin, a RING-between-RING-type E3 ubiquitin ligase associated with Parkinson's disease, has a wide neuroprotective activity, preventing cell death in various stress paradigms. We identified a stress-protective pathway regulated by parkin.."

So presumably the +ve reg annotation should be deleted

**Comment by ValWood** [comment]
had a quick look at the paper because I was curious about the human annotations.

1. There are a number of TAS to cytokinesis , but cytokinesis is not mentioned in this paper?

2. If SPAST is involved in spindle disassembly (I don't know if it is)

SPAST | Spastin |   | spindle disassembly | part_ofcytoskeleton-dependent cytokinesis | ARUK-UCL | Homo sapiens | TAS

spindle assembly isn't part of cytokinesis, it occurs at the same time? (during mitotic telophase?)


3. I don't think Lem is involved in  "protein localization to chromatin"

LEMD2 | LEM domain-containing protein 2 |   | protein localization to chromatin | has inputUniProtKB:Q8WUX9happens_duringanaphase more... | ARUK-UCL | Homo sapiens | IMP

IIRC it serves as a nuclear membrane attachment site for chromatin


4. 
escrt-iii_human | ESCRT-III complex |   | actomyosin contractile ring contraction | part_ofcytoskeleton-dependent cytokinesis | ARUK-UCL | Homo sapiens | TAS

If ESCRT-II *is* involved in cytokinesis ring contraction (I don't think it is? it might have some role in cell separation after the ring has constricted?) ...

but *If  it was* you don't need a part of extension to "cytoskeleton-dependent cytokinesis" because  "cytokinesis ring contraction" is a child of  "cytoskeleton-dependent cytokinesis" 

5. Delete,  has nothing to do with membrane permeability:

  | escrt-iii_human | ESCRT-III complex |   | negative regulation of nuclear envelope permeability |   | ARUK-UCL | Homo sapiens | TAS

**Comment by ValWood** [comment]
This paper is about repairing breaches in the newly formed nuclear envelope, or when gaps in the nuclear envelope are made for example to insert the spindle pole body.

there is a nice cartoon in the supp:

https://cloud.githubusercontent.com/assets/7359272/25667564/c4a21e46-301b-11e7-8471-b54048d3976b.jpg

This  will hopefully makes it clearer why it has nothing to do with "membrane permeability"...look at the topology. It's a bit like the "nucleocytoplasmic transport is  not transmembrane transport" issue.

**Comment by RLovering** [comment]
Hi Antonia

thank you for spotting this error. I have updated the annotation to the neg term, I think this emphasises why it is sensible to look at a whole protein record, when annotating. Or Rebecca might have just copied the wrong ID and moved on too quickly.

Thanks again

Ruth

**Comment by RLovering** [comment]
Hi Val

We will look at these annotations, as there are so many it will take a while, but we will follow this up. 

See you soon

Ruth

**Comment by ValWood** [comment]
assigning to Ruth.

**Comment by BarbaraCzub** [comment]
Hi @ValWood 

Following discussions and advice from @rachhuntley and @RLovering I have removed all of ARUK-UCL annotations made to the _pombe_ gene products.

I have kept the 'cytokinesis' annotations and the citations in the comments, as I think they provide relevant support. However, following @ValWood ’s question I have updated the 'spindle disassembly’ to 'microtubule severing' (AE part_of cytoskeleton-dependent cytokinesis, occurs_in midbody, occurs_in polar microtubule). The microtubules present at the midbody are in fact the spindle microtubules/'polar microtubules'. I additionally included a few more TAS annotations with supporting statements in the comments fields.

I kept the 'protein localization to chromatin' annotations, as the data support them. E.g. "siRNA depletion of CHMP7 abrogated robust IST1/CHMP8 and CHMP2A recruitment to chromatin, as assayed by detection of endogenous protein by fixed-cell immunofluorescence (Fig. 7 B and D,)”. I also made a few more cellular component annotations using the 'colocalises_with' qualifier.

Thanks,
Barbara
